client/less/main.less

```diff
@@ -1071,6 +1071,21 @@ code {
       background-color: #666;
     }
   }
+  .hamburger-dropdown {
+    .dropdown .dropdown-menu,
+    .open #nav-Community-dropdown {
```

Comment:
I think the selectors could be condensed a bit. The selector for the `a` inside of this would be 
```css
.night .hamburger-dropdown .dropdown .dropdown-menu a
```
Everything seemed to work the same without `.dropdown`.
`.hamburger-dropdown` was only required for the link hovering section.

That makes the selector for the same `a`
```css
.night .dropdown-menu a
```
While keeping the same amount of lines of code.